PicaSWB.js

```diff
@@ -41,6 +41,8 @@
 
 var ssgNummer = "1";
 var defaultLanguage = "eng";
+var physicalForm = "A";//0500 Position 1
```

Comment:
habe hier den Wert in "O" geändert und das Ergebnis dementsprechend richtig:Feld 2051 und 4085 ausgegeben.